jest-preprocess.js

```diff
@@ -0,0 +1,5 @@
+const babelOptions = {
+    presets: ["babel-preset-gatsby"],
+  }
```

Comment:
Sorry to be _ultra_ annoying here, but there's a few spots where the closing brace has a couple of spaces before it. If you wouldn't mind removing that extra whitespace I'd appreciate it! I'll note the other spots for clarity :)